lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -377,6 +379,36 @@ class DetailsRenderer {
     return element;
   }
 
+  /**
+   * @param {LH.Audit.Details.SourceLocationValue} item
+   * @return {Element|null}
+   * @protected
+   */
+  renderSourceLocation(item) {
+    if (!item.url) {
```

Comment:
I'm guarding against an empty string here. Perhaps that's not necessary, and just allowing the url to be meaningless is better than showing nothing.

It may be possible to get an empty string from `stylesheet.sourceURL` if it came from a comment - I'd have to check. It is certainly possible for a plugin to provide an empty string.